StyleGuide.md

```diff
@@ -1365,6 +1366,49 @@ Considerations:
 - If you intend to link to a header present on two or more tabsets on the same page, the header targets must be uniquely named. If you require identical header names, use explicit, unique HTML anchor names for each (in form `<a name="uniquename"></a>` as shown under [Links](#links).
 - For the first-defined tab, specifying its `filter` value in a link is functionally the same as omitting it. For all other tabs, the explicit filter name is required. You can think of this first tab as the "default" tab in this context: if not otherwise specificed, Jekyll will always open with the first tab's contents displayed.
 
+### Comments
+
+There may be situations that require adding an explanation below a piece of content or to temporarily suppress content within a page. In those situations, use [Liquid comments](https://shopify.github.io/liquid/tags/template/#comment).
+
+Page source:
+
+```
+This sentence is visible!
+{% comment %}
+This sentence is not visible!
+{% endcomment %}
+This sentence is also {% comment %}not {% endcomment %}visible.
```

Comment:
I think only the word "not" is not visible, right? Does it really hide a whole line that has a comment in the middle of it?

Reply:
```
{% comment %}
This sentence is not visible!
{% endcomment %}
```
The paragraph containing `This sentence is not visible!` gets suppressed. In line 1380, only the word `not` and the space following it is suppressed. It doesn't hide the whole line. It only hides what is strictly in between the `{% comment %}` and `{% endcomment %}` tags.